context/requirements.txt

```diff
@@ -4,3 +4,5 @@ requests==2.22.0
 pyyaml==5.2
 python-datauri==0.2.8
 python-frontmatter==0.5.0
+# COMMONS_BRANCH controls the branch of hubmap-commons.
+git+git://github.com/hubmapconsortium/commons.git@${COMMONS_BRANCH}#egg=hubmap-commons
```

Comment:
This makes me very uncomfortable. I would much prefer to have a pinned version, so that builds will be reproducible.

Reply:
Agreed, and it is now pinned to a specific release.